### PR TITLE
NCBI Exports replace DataTables with Ant Design Table

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

Removed DataTables and updated with Ant Design Tables for the NCBI Export Tables.

## Related issue

N/A

## Checklist

Things for the developer to confirm they've done before the PR should be accepted:

*   [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
*   [x] Tests added (or description of how to test) for any new features.
~~*   [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/webapp/resources/js/components/ncbi/export-table/NcbiExportTable.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    getNCBIExports({ url }).then(setExports);
+  }, [url]);
+
+  const columns = [
+    {
+      title: i18n("NcbiExportTable.id"),
+      dataIndex: "id",
+      render(id, item) {
+        return (
+          <a className="t-biosample-id" href={`${window.location.href}/${id}`}>
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/ncbi/export-table/NcbiExportTable.jsx:21]: Is there a better way to set the href then using the window.location.href?

File: src/main/webapp/resources/js/apis/ncbi/ncbi.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export async function getNCBIExports({ url }) {
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/apis/ncbi/ncbi.js:1]: File and function comments

File: src/main/java/ca/corefacility/bioinformatics/irida/service/impl/export/NcbiExportSubmissionServiceImpl.java [hunk 3]
```diff
@@ -102,7 +103,13 @@ public NcbiExportSubmission create(NcbiExportSubmission object) throws Constrain
 			throws IllegalArgumentException {
 		return super.list(page, size, order, sortProperties);
 	}
-	
+
```
Comment (deepsidhu85) [inline src/main/java/ca/corefacility/bioinformatics/irida/service/impl/export/NcbiExportSubmissionServiceImpl.java:106]: Add method comment. Probably just @inheritDoc

Comment (joshsadam) [reply src/main/java/ca/corefacility/bioinformatics/irida/service/impl/export/NcbiExportSubmissionServiceImpl.java:106]: Updated in 4af6a7e

File: src/main/webapp/resources/js/apis/ncbi/ncbi.js (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/apis/ncbi/ncbi.js]: Updated in 4af6a7e

Comment (joshsadam) [reply src/main/webapp/resources/js/apis/ncbi/ncbi.js]: Updated in 4af6a7e

File: src/main/webapp/resources/js/components/ncbi/export-table/NcbiExportTable.jsx (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/ncbi/export-table/NcbiExportTable.jsx]: Updated in 4af6a7e

File: src/main/webapp/resources/js/pages/projects/ncbi-export/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/projects/ncbi-export/index.js:1]: Updated in 4af6a7e

File: src/main/webapp/resources/js/components/ncbi/upload-states/NcbiUploadStates.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/components/ncbi/upload-states/NcbiUploadStates.jsx:1]: Updated in 4af6a7e

File: src/main/webapp/resources/js/components/ncbi/export-table/NcbiExportTable.jsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply src/main/webapp/resources/js/components/ncbi/export-table/NcbiExportTable.jsx:1]: Updated in 4af6a7e

**Pull request merged**